dev/com.ibm.ws.kernel.metatype.helper/src/com/ibm/wsspi/config/internal/FilesetImpl.java

```diff
@@ -368,6 +368,35 @@ private void getHumanReadableSummary(StringBuilder sb) {
             }
         }
 
+       public boolean hasFilters() {
+            return !includes.isEmpty() && excludes.isEmpty();
```

Comment:
```suggestion
            return !includes.isEmpty() && !excludes.isEmpty();
```
I assume this was wrong.